assets/components/forms/formHOCs/withArrow.scss

```diff
@@ -0,0 +1,9 @@
+.component-with-arrow {
+  .svg-arrow-right-straight {
+    height: 45%;
+    display: inline-block;
+    vertical-align: top;
+    margin-left: 16px;
```

Comment:
can these be multiples of `$gu-h-spacing`?

Reply:
I'm not sure it's worth it here, this would be `$gu-h-spacing * 0.8`.

Reply:
why is it `16` though? could it it be a rounder 15?